docs/setup-guide/arktos-with-mizar-cni.md

```diff
@@ -22,22 +22,17 @@ If this machine will be used as the master of a multi-node cluster, please set a
 2. Install the mizar dependencies
 ```bash
 wget https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/bootstrap.sh
-# If kernel version is less than 5.6 then run the following line else skip
+# If kernel version is less than 5.6 then download the kernelupdate.sh else skip
 wget https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/kernelupdate.sh  
-# If using docker-ce in the environment then edit the bootstrap file and remove the docker.io package. 
+# If using docker-ce in the environment then edit the bootstrap.sh and remove the docker.io package before running. 
```

Comment:
Why do you want to introduce manual steps? I have mentioned many times before that we need to minimize manual steps. Can you guys understand English? You guys just don't get it, do you?